src/components/Nav/Nav.js

```diff
@@ -1,7 +1,49 @@
-import './Nav.scss';
-
-function Nav() {
-  return <></>;
-}
-
-export default Nav;
+import './Nav.scss';
+import '../../styles/variables.scss';
+import { Link } from 'react-router-dom';

```

Comment:
```suggestion
import { Link } from 'react-router-dom';
import './Nav.scss';
import '../../styles/variables.scss';
```
import 순서 컨벤션에 따라 수정해주세요!